src/modules/commander/commander_params.c

```diff
@@ -1049,48 +1049,49 @@ PARAM_DEFINE_FLOAT(COM_SPOOLUP_TIME, 1.0f);
  * A warning is triggered if the currently estimated wind speed is above this value.
  * Warning is sent periodically (every 1min).
  *
- * A negative value disables the feature.
+ * Set to 0 to disable.
  *
- * @min -1
- * @max 30
+ * @min 0
  * @decimal 1
  * @increment 0.1
  * @group Commander
  * @unit m/s
  */
-PARAM_DEFINE_FLOAT(COM_WIND_WARN, -1.f);
+PARAM_DEFINE_FLOAT(COM_WIND_WARN, 0.f);
 
 /**
  * Maximum allowed flight time
  *
  * The vehicle aborts the current operation and returns to launch when
  * the time since takeoff is above this value. It is not possible to resume the
- * mission or switch to any mode other than RTL or Land.
+ * mission or switch to any auto mode other than RTL or Land. Taking over in any manual
+ * mode is though still possible.
  *
- * Set a negative value to disable.
+ * There will be a warning message sent every 1 minute with updated
+ * remaining time till RTL once passed the 90% threshold and until RTL is triggered.
  *
+ * Set to 0 to disable.
  *
  * @unit s
- * @min -1
- * @max 10000
- * @value 0 Disable
+ * @min 0
  * @group Commander
  */
-PARAM_DEFINE_INT32(COM_FLT_TIME_MAX, -1);
+PARAM_DEFINE_INT32(COM_FLT_TIME_MAX, 0);
 
 /**
  * Wind speed RLT threshold
  *
- * Wind speed threshold above which an automatic return to launch is triggered
- * and enforced as long as the threshold is exceeded.
+ * Wind speed threshold above which an automatic return to launch is triggered.
+ * It is not possible to resume the mission or switch to any auto mode other than
+ * RTL or Land once exceeded once. Taking over in any manual
+ * mode is though still possible.
  *
- * A negative value disables the feature.
+ * Set to 0 to disable.
  *
- * @min -1
- * @max 30
+ * @min 0
  * @decimal 1
  * @increment 0.1
  * @group Commander
  * @unit m/s
  */
-PARAM_DEFINE_FLOAT(COM_WIND_MAX, -1.f);
+PARAM_DEFINE_FLOAT(COM_WIND_MAX, 0.f);
```

Comment:
I would honestly keep a negative value to indicate as clearly as possible the value is not used.